ansible/copy_files_from_ctrl_node.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
I can't see any evidence that this script is used - possibly this was temporarily used but replaced by a step in your playbook? If so, I think we should be able to delete this!

Reply:
I will delete it, it was only to make the process to git commit easy